### ISSUE TITLE
Controls Within “Music” Tree Item Are Not Keyboard Accessible: A11y_Microsoft MakeCode_New Project_Editor_Keyboard

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/pxt-microbit#6862]
"Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Feng.ms%2Fdocs%2Fcoreai%2Fdevdiv%2Fone-engineering-system-1es%2F1es-docs%2Fes-chat%2Findex&data=05%7C02%7Cv-mchougule%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838215421556%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=%2B1yeHqbNhHcMuQaZNXDo%2B%2FiWxH0mZFSmWpamQk2ftws%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2FCAI%2520Accessibility%2520Support%2520Channel%3FgroupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7C02%7Cv-mchougule%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838215450064%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=yncZXSKXaJlQ3HpXeR3TK03MNIhUe8tV2yZl49clFZY%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Foutlook.office.com%2Fbook%2Fmicrosoft2779007%2540microsoft.onmicrosoft.com%2Fs%2FbLeGBltMW0qI0WRLsTqx2A2%3Fismsaljsauthenabled&data=05%7C02%7Cv-mchougule%40microsoft.com%7C26cff27270c448f48a0e08de3f569146%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639017838215467554%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=i4Fn6qPJw4eg3Ny6En0%2FPnIqw%2BpJpHuT%2F%2BZH2hE%2FRTA%3D&reserved=0)."

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

"Contact [CAI Accessibility Support Channel](https://teams.microsoft.com/l/channel/19:DeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%40thread.tacv2/CAI%20Accessibility%20Support%20Channel?groupId=f5eb428c-9e6e-4c16-8143-33fe3961a9ae&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) for Accessibility queries on design, development, test, usability or any processes"

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### GitHub Tags:
#A11yTCS; #A11yMAS; #Win11; #DesktopWeb; #Microsoft MakeCode; #E2E_MicrosoftMakeCode_Web_May2026; #FTP; #ChromiumEdge; #Keyboard; #A11ySev1; #WCAG2.1.1; #AILimited; #ES Chat;

### Environment Details:
Application Name: Microsoft MakeCode
URL: https://makecode.microbit.org/beta
windows Edition: Windows 11 Enterprise Insider Preview(24H2)
OS Build: 26544.1000

### Repro Steps:
1. Launch application URL: https://makecode.microbit.org
2. Home page will open.
3. Tab to "New Project" option and press enter.
4. Editor screen will open.
5. Tab to "Basic, Input, Music, Led, more, Radio" controls and select it.
6. Tab through all the controls appearing after selection of element.
7. Observe the issue.

**Note:** A similar issue has been noticed across all the trees in the editor section, such as Basic, Input, LED, Radio, Loops, and Logic.

### Actual:
When navigating to the “Music” tree item, the user is unable to access or interact with the controls (e.g., melody input, tempo, tone, dropdowns) using the keyboard. The focus does not move into the controls within the selected tree item.

### Expected result:
Once the “Music” tree item is selected, all interactive controls within the section should be reachable and operable using keyboard navigation in a logical order (using Tab/Arrow keys as applicable).

### User impact:
Keyboard-only users and screen reader users are unable to access or interact with the controls inside the “Music” section, preventing them from using essential functionality.

### Attachment:

https://github.com/user-attachments/assets/46a579a2-4f60-48b9-9dcd-a4b4f7d6f818

**Comment by riknoll** [comment]
you don't have keyboard mode enabled. this setting is now on by default and this bug is fixed
